### ISSUE TITLE
Add Post creation in topic threads

### ISSUE THREAD

**Issue by jean1880** [wendelltron/tekapp#14]
#### User Story
As a user, I want to be able to posts to topic threads, to participate in discussions

====
#### Success Criteria
- [ ] Button to reply to threads
- [ ] Form to write out post (should have advanced editor, like [this] (http://xing.github.io/wysihtml5/)
- [ ] Form should detect @ symbol, to link users to post
- [ ] Should only be enabled/visible if user is logged in

**Comment by jean1880** [comment]
Check out bootstrap version of [wysihtml5](http://jhollingworth.github.io/bootstrap-wysihtml5/), might work better

**Comment by DaAwesomeP** [comment]
We could try to implement the same editor: https://meta.discourse.org/t/looking-editor-source/3150
My favorite is CkEditor, but anything is really fine.

**Comment by jean1880** [comment]
Yeah, i'm fine with whatever, I was just quickly throwing some options out, I'm not picky, just as long as it can handle the html creation, for URLs etc on the fly, makes our jobs nice and easy
